op-node/rollup/types_test.go

```diff
@@ -27,14 +28,23 @@ func randConfig() *Config {
 			L1:     eth.BlockID{Hash: randHash(), Number: 424242},
 			L2:     eth.BlockID{Hash: randHash(), Number: 1337},
 			L2Time: uint64(time.Now().Unix()),
+			SystemConfig: eth.SystemConfig{
+				BatcherAddr: randAddr(),
+				Overhead:    randHash(),
+				Scalar:      randHash(),
```

Comment:
If these are used in tests that have fees, could see some weird results if random hashes are used as the values

Reply:
It's only a basic json encoding test, the values don't matter that much in this case. We can fuzz the rollup.json encoding later, but it seems low risk since we only deal with one such config in a network.